### DISCUSSION TITLE
[HW Accel Support]: Failed to play recordings (error 3): CHUNK_DEMUXER_ERROR_APPEND_FAILED: Failed to prepare video sample for decode

### DISCUSSION THREAD

**Discussion by moogoos** [blakeblackshear/frigate#15062] [Hardware Acceleration Support] [answered]
### Describe the problem you are having

Failed to play recordings (error 3): CHUNK_DEMUXER_ERROR_APPEND_FAILED: Failed to prepare video sample for decode

### Version

15beta

### Frigate config file

```yaml
NA
```

### docker-compose file or Docker CLI command

```yaml
NA
```

### Relevant Frigate log output

```shell
No logs, just an error in the client. No errors in logs.
```

### Relevant go2rtc log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Install method

Docker Compose

### Object Detector

OpenVino

### Network connection

Wired

### Camera make and model

Hikvision

### Screenshots of the Frigate UI's System metrics pages

![Screenshot 2024-11-18 at 8 03 31 PM](https://github.com/user-attachments/assets/0efe6f35-a5c6-4d20-b110-b1af17b96ffe)

### Any other information that may be helpful

All my cameras are hibvision and working fine. I have been using frigate since version .11 I think maybe, .12. After about 3 days after starting over with a new db and deploy, these errors start appearing for some of the cameras, not all of them when i try and review history. 

Errors on both iOS and Mac OS X Chrome.

**Comment by moogoos** [comment]
And now it’s working fine. This is odd. Any idea what this error message eludes too? Any experience with it? Is my GPU having intermittent issues?

**Comment by hawkeye217** [comment, marked as answer]
This is a client error typically specific to Chrome and chromium based browsers, which are often more particular about bad data in a video stream. 

It's hard to say exactly what in your setup caused it, but if it happens again, you might try another browser to see what, if any, errors are produced there.
